Documentation/DataSources/DataSources.md

```diff
@@ -53,12 +53,14 @@ To make a query control use this data source, use the _Data source_ drop down to
 
 ![A image of a workbook with ARG data](../Images/ArgDataSource.png)
 
-## Alerts
-Workbooks allow users to visualize the active alerts related to their resources. This allows the creation of reports that bring together notification data (alert) and diagnostic information (metrics, logs) into one report. This information can also be joined together to create rich reports that combines insights across these data sources. 
+## Alerts (classic)
```

Comment:
nit: you added classic here but the dropdown still says 'preview' which might confuse people. Maybe leave it as Alerts (preview) but highlight the suggested template